src/index.js

```diff
@@ -143,6 +144,34 @@ export default function ({ template, types }) {
             }
           },
         });
+
+        programPath.scope.crawl();
```

Comment:
Let's keep it simple for now with `programPath.scope.crawl();` 👍 .
Just what about moving the call in the if statement?